features/automation/optimization/autoTakeProfit/controls/AutoTakeProfitFormControl.tsx

```diff
@@ -26,28 +34,59 @@ export function AutoTakeProfitFormControl({
   isAutoTakeProfitActive,
   tokenMarketPrice,
   vault,
+  autoTakeProfitTriggerData,
+  ethMarketPrice,
+  shouldRemoveAllowance,
+  txHelpers,
 }: AutoTakeProfitFormControlProps) {
   const [autoTakeProfitState] = useUIChanges<AutoTakeProfitFormChange>(AUTO_TAKE_PROFIT_FORM_CHANGE)
-
+  const isAddForm = true
   const feature = AutomationFeatures.AUTO_TAKE_PROFIT
-  const { closePickerConfig, min, max } = getAutoTakeProfitStatus({
+  const { closePickerConfig, min, max, resetData } = getAutoTakeProfitStatus({
     autoTakeProfitState,
     tokenMarketPrice,
     vault,
   })
+  const { addTxData, textButtonHandlerExtension } = getAutoTakeProfitTxHandlers({
+    vaultData: vault,
```

Comment:
I know that is really nit-picking, but why not just
```suggestion
    vault,
```
I checked, Constant Multiple has it like that.

Reply:
ok